packages/tools/src/utilities/stackPrefetch/stackContextPrefetch.ts

```diff
@@ -222,7 +222,7 @@ function prefetch(element) {
 
   const useNativeDataType = useNorm16Texture || preferSizeOverAccuracy;
 
-  indicesToRequestCopy.forEach((imageIdIndex) => {
+  stackPrefetch.indicesToRequest.forEach((imageIdIndex) => {
```

Comment:
Thanks for your pull request (PR). I tested your test code in the `main` branch, and it failed, as you mentioned. I also tried your proposed solution, and it passed - good work.

However, I reverted the change on line 255, and the tests still passed. What was the reason for modifying that line?

Reply:
The indicesToRequestCopy is the original set of images, including things that are already in cache.  If you don't use the correct set, then you will end up re-fetching already cached images.  The line is correct as changed.

Reply:
I'm thinking about how to test this, but without quite a bit of mocking, I don't see a good way of testing it.